sdcm/utils/raft/__init__.py

```diff
@@ -26,6 +26,21 @@ class Group0MembersNotConsistentWithTokenRingMembersException(Exception):
     pass
 
 
+class NodeState(StrEnum):
+    BOOTSTRAPPING = "BOOTSTRAPPING"
+    DECOMMISSIONING = "DECOMMISSIONING"
+    NORMAL = "NORMAL"
+    LEAVING = "LEAVING"
+    SHUTDOWN = "shutdown"
```

Comment:
Why on UPPERCASE as the rest?

Reply:
Added all other statuses(they not used in this commit, but could be used in future)

Reply:
i checked them with Emil, he give me all statusses.